site/src/atomics/border.md

```diff
@@ -58,23 +58,28 @@ Here is an example of applying border to specific side of the element:
 
 ### Radius
 
-Currently two classes are available to set border radius:
+There are a few classes available to set border radius:
 
-| class              | size      |
-| ------------------ | --------- |
-| `border-radius`    | `0.25rem` |
-| `border-radius-lg` | `0.5rem`  |
+| class                | size            |
+| -------------------- | --------------- |
+| `border-radius-sm`   | `0.125rem`      |
+| `border-radius`      | `0.25rem`       |
+| `border-radius-lg`   | `0.375rem`      |
+| `border-radius-full` | `full rounding` |
```

Comment:
border-radius-circular ?

Reply:
I need to find out how to make polls for the new classes 🤦🏻‍♀️ I don't think I like the `-circular`, it's a bit confusing to me and quite hard to remember it means completely rounded corners. Not feeling strong about `-full` either tbh. In docs we have an `is-rounded` class that does relevant modification, but `border-radius-rounded` doesn't give me an association that it's doing the full rounding.

@wibjorn @homing1 @BenDMyers any thoughts/ideas would be appreciated 🙏🏻

Reply:
Just to throw out a third possibility... maybe `border-radius-max`?

Reply:
I think `border-radius-rounded` to say consistent with `is-rounded` would be my favorite bet.

Reply:
I actually like the `-max` option! Let's see what others think about it.

Reply:
`rounded` is the way it has been and the way it should stay I think; less work all around and a good name

Reply:
round, rounded, or circular sound good to me.

Reply:
Agreed @jdanyow

Reply:
updated to `border-radius-rounded` since that sounded like a preferred option :D